### PR TITLE
Move slow-grad checks to CUDA-11.6

### PR DESCRIPTION
**Pull request by atalman**
Mitigates #84192 by skipping two tests

Please note: We tried to increase the tolerance for test_fn_gradgrad_linalg_det_singular_cuda_float64 but this did not help.
Ref:
Increase `test_fn_gradgrad_linalg_det_singular_cuda_float64` error tolerance to  1e-4 as suggested in https://github.com/pytorch/pytorch/issues/84192#issuecomment-1230644574

### CODE REVIEW

File: torch/testing/_internal/opinfo/definitions/linalg.py [hunk 1]
```diff
@@ -1276,6 +1276,11 @@ def make_input():
                 "test_fn_fwgrad_bwgrad",
                 dtypes=(torch.complex128,),
             ),
+            DecorateInfo(
+                "TestGradients",
+                "test_fn_gradgrad",
```
Comment (malfet) [inline torch/testing/_internal/opinfo/definitions/linalg.py:1281]: Don't you want to limit tolerance reduction to just cuda tests?
```suggestion
                "test_fn_gradgrad",
                device="cuda",
```

**Review by malfet — APPROVED** [review]

**Review by huydhn — APPROVED** [review]
Linter, otherwise LGTM!

File: torch/testing/_internal/opinfo/definitions/linalg.py [hunk 2]
```diff
@@ -1276,6 +1276,12 @@ def make_input():
...
+                device="cuda",
+                toleranceOverride({torch.float64: tol(atol=1e-4, rtol=1e-4)})
```
Comment (atalman) [inline torch/testing/_internal/opinfo/definitions/linalg.py:1283]:
```suggestion
                toleranceOverride({torch.float64: tol(atol=1e-4, rtol=1e-4)}),
```

**Review by lezcano — APPROVED** [review]
the linalg part LGTM!

File: torch/testing/_internal/opinfo/definitions/linalg.py [hunk 3]
```diff
@@ -1276,6 +1276,18 @@ def make_input():
                 "test_fn_fwgrad_bwgrad",
                 dtypes=(torch.complex128,),
             ),
+            DecorateInfo(
+                unittest.skip("Skipped!"),
```
Comment (malfet) [inline torch/testing/_internal/opinfo/definitions/linalg.py:1280]:
```suggestion
                unittest.skip("Skipped, see https://github.com/pytorch/pytorch/issues/84192"),
```

**Review by malfet — CHANGES_REQUESTED** [review]
Please update PR description (namely remove "Fixes" statement, as it does not fix the problem, but rather disables those tests)
And reference the issue in test skip reason

File: torch/testing/_internal/opinfo/definitions/linalg.py [hunk 4]
```diff
@@ -1276,6 +1276,18 @@ def make_input():
...
                 dtypes=(torch.complex128,),
             ),
+            DecorateInfo(
+                unittest.skip("Skipped, see https://github.com//issues/84192"),
+                "TestGradients",
+                "test_fn_gradgrad",
+                device_type="cuda",
+            ),
+            DecorateInfo(
+                unittest.skip("Skipped!"),
```
Comment (atalman) [inline torch/testing/_internal/opinfo/definitions/linalg.py:1286]:
```suggestion
                unittest.skip("Skipped, see https://github.com//issues/84192"),
```

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**